srcpkgs/elog/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'elog'
+pkgname=elog
+version=3.1.4.1
+revision=1
+_distver="${version:0:-2}-${version:(-1)}"
+wrksrc="$pkgname-${version:0:-2}"
+build_style=gnu-makefile
+make_dirs="/var/lib/elog/logbooks 0750 elog elog"
+conf_files="/etc/elog/elogd.cfg"
+system_accounts="elog"
+short_desc="Logbook system to manage notes through a Web interface"
+maintainer="Christian Poulwey <christian.poulwey@t-online.de>"
+makedepends="libressl-devel"
+license="GPL-3.0-or-later"
+homepage="https://elog.psi.ch/elog/"
+distfiles="https://elog.psi.ch/elog/download/tar/$pkgname-$_distver.tar.gz"
+checksum=4bfa2bef6cfa20aba73c9c6c044a4c83f20cf03e7de3c0dda4a4477f28b7b72e
+
+do_build() {
+	make CC="$CC" LD="$LD" AR="$AR" \
+			RANLIB="$RANLIB" OBJDUMP="$OBJDUMP" \
+			CFLAGS=" -Imxml $CFLAGS" LDFLAGS="$LDFLAGS" \
+			all
+}
```

Comment:
Why are you overwriting this at all?

Reply:
There needs to be "-Imxml" set in the CFLAG variable and I did not find an option to do so in the gnu_makefile build style

Reply:
just add CFLAGS="-lmxml" to the top-level of the template